src/metabase/channel/email/messages.clj

```diff
@@ -297,69 +292,6 @@
       :below)
     :rows))
 
-(defn- first-card
-  "Alerts only have a single card, so the alerts API accepts a `:card` key, while pulses have `:cards`. Depending on
-  whether the data comes from the alert API or pulse tasks, the card could be under `:card` or `:cards`"
-  [alert]
-  (or (:card alert)
-      (first (:cards alert))))
-
-(defn common-alert-context
-  "Template context that is applicable to all alert templates, including alert management templates
-  (e.g. the subscribed/unsubscribed emails)"
-  ([alert]
-   (common-alert-context alert nil))
-  ([alert alert-condition-map]
-   (let [{card-id :id, card-name :name} (first-card alert)]
-     (merge (common-context)
-            {:emailType                 "alert"
-             :questionName              card-name
-             :questionURL               (urls/card-url card-id)
-             :sectionStyle              (channel.render/section-style)}
-            (when alert-condition-map
-              {:alertCondition (get alert-condition-map (pulse->alert-condition-kwd alert))})))))
-
-(defn- schedule-hour-text
-  [{hour :schedule_hour}]
-  (.format (LocalTime/of hour 0)
-           (DateTimeFormatter/ofPattern "h a")))
-
-(defn- schedule-day-text
-  [{day :schedule_day}]
-  (get {"sun" "Sunday"
-        "mon" "Monday"
-        "tue" "Tuesday"
-        "wed" "Wednesday"
-        "thu" "Thursday"
-        "fri" "Friday"
-        "sat" "Saturday"}
-       day))
-
-(defn- schedule-timezone
-  []
-  (or (driver/report-timezone) "UTC"))
-
-(defn notification-card-schedule-text
```

Comment:
this is the old implementation. the new implementation is in `metabase.channel.shared`